S100SapCallback_OpenApi_Spec.yml

```diff
@@ -0,0 +1,151 @@
+openapi: 3.0.1
+info:
+  title: UKHO S100 ERP Facade Service API
+  description: |
+    The UKHO S100 ERP Facade Service API handles callback event received from SAP.
+  contact:
+    name: UKHO
+    email: contact@ukho.co.uk
+  version: 0.0.1
+servers:
+  - url: https://erpfacade-dev-api.azurewebsites.net
+    description: Dev URL
+  
+paths:
+  /s100erpfacade/callback/{correlationId}:
+    get:
+      description: Returns response status for the callback request from SAP.
+      tags:
+        - "S100 ERP Facade"
+      operationId: s100erpfacade
+      parameters:
+      - name: correlationId
+        in: path
```

Comment:
If this is a POST operation, this should be passed in the body.

Reply:
Updated correlationId to be passed in the body. Thanks